app/javascript/packages/verify-flow/verify-flow-step-indicator.tsx

```diff
@@ -0,0 +1,73 @@
+import { StepIndicator, StepIndicatorStep, StepStatus } from '@18f/identity-step-indicator';
+import { t } from '@18f/identity-i18n';
+
+// i18n-tasks-use t('step_indicator.flows.idv.getting_started')
+// i18n-tasks-use t('step_indicator.flows.idv.verify_id')
+// i18n-tasks-use t('step_indicator.flows.idv.verify_info')
+// i18n-tasks-use t('step_indicator.flows.idv.verify_phone_or_address')
+// i18n-tasks-use t('step_indicator.flows.idv.secure_account')
+
+/**
+ * Mapping of flow form steps to corresponding step indicator step.
+ */
+const FLOW_STEP_STEP_MAPPING: Record<string, string> = {
```

Comment:
if we did the enum/union type:
```suggestion
const FLOW_STEP_STEP_MAPPING: Record<string, StepIndicatorStep> = {
```

Reply:
Updated in d16b70405.